dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/RestMetricsTest.java

```diff
@@ -205,6 +206,7 @@ public void testAbort() throws IOException {
         ensureEmptyMonitorStats(ABORT_METHOD_INDEX);
     }
 
+    @SkipForRepeat("EE10_FEATURES") //Skipping until MP Metrics is upgraded for MP 6.0
```

Comment:
we should open an issue for this

Reply:
https://github.com/OpenLiberty/open-liberty/issues/22519